### DISCUSSION TITLE
change menu container background color

### DISCUSSION THREAD

**Discussion by AlirezaBabaeii** [adobe/react-spectrum#3832] [General]
How to change menu container background color? 

![Screenshot from 2022-12-11 12-36-11](https://user-images.githubusercontent.com/99098510/206895374-9c87f9d7-45b5-4122-b76d-ea815955e50d.png)

**Comment by snowystinger** [comment]
We do not support changing the background of the menu, it has been designed to meet certain minimum contrast ratios. Same thing with the background on the button, and with the background of the container in which the button has been placed. You could look at the themes we have, maybe one of those will suit you better. https://react-spectrum.adobe.com/react-spectrum/theming.html#how-themes-are-defined
